sklearn/tree/_export.py

```diff
@@ -683,19 +687,24 @@ def export(self, decision_tree, ax=None):
     def recurse(self, node, tree, ax, max_x, max_y, depth=0):
         import matplotlib.pyplot as plt
 
-        kwargs = dict(
-            bbox=self.bbox_args.copy(),
+        # kwargs for annotations without a bounding box
+        non_box_kwargs = dict(
             ha="center",
```

Comment:
```suggestion

```

Remove the line here in order to specify different horizontal alignment for True/False labels later. Instead, we move `ha="center"` to `kwargs`.

Reply:
Done